src/components/NFTPage/PreviewCard.vue

```diff
@@ -68,9 +68,10 @@
       <ButtonV2
         v-if="url"
         preset="outline"
-        :text="$t('campaign_nft_item_view_details_label')"
-        :href="url"
+        :text="isContentViewable ? $t('nft_details_page_button_view') : $t('nft_details_page_button_collect_to_view')"
```

Comment:
Suggest adding `w-full` class for view buttons
Look very bad when there are multiple buttons with different widths
![image](https://user-images.githubusercontent.com/17264716/230257337-7aa1ced0-f628-4442-aea6-88e54475b14e.png)